### ISSUE TITLE
Blacklist of words to never buttify

### ISSUE THREAD

**Issue by pennstatephil** [ducktrshessami/discord-buttsbot#50]
It'd be nice to have a user-configurable list of words to never buttify. Words like "to", "and", etc tend to produce less funny results when buttified so it'd be nice to blacklist them and have buttsbot pick a different word instead.

**Comment by ducktrshessami** [comment]
I personally disagree with this idea since I feel the chance to produce incoherent nonsense is what makes the semi-coherent messages special. Which is to say, I've always felt the chaos was part of the experience. It's like how people would go nuts when [ShitpostBot 5000](https://knowyourmeme.com/memes/sites/shitpostbot-5000/) created an actually coherent meme. If it was actually tailored to do so, it'd just be a content farm; it producing good content wouldn't really have meaning.

That being said, I like things having customization, and I can also imagine scenarios like tiny, single-channel servers for small friend-groups where you wouldn't ignore the whole channel but would want some sensitive topics ignored.

So I'll probably add this anyways when I have time (probably in the next week or two).

**Comment by pennstatephil** [comment]
Thank you for adding this! When should it be available for use? (Not sure what the roll-out process is)

**Comment by ducktrshessami** [comment]
I'm currently updating some other things with the host first, but probably within an hour or so.

Also while you're here, I might as well mention: due to how the buttification works (directly matches alphabetical characters), currently only alphabetical characters are accepted for word-ignores, so things like contractions still can't be ignored. Not sure if that affects how you were intending to use it, but just thought I'd give a heads-up.

TLDR on why: I'd need to rewrite how I parse syllables in order to change this, and it might involve reducing its ability to handle non-English words.

**Comment by pennstatephil** [comment]
No, I was mainly looking to ignore prepositions... buttifying those are generally not funny :(
I may end up trying to submit a PR to add `/ignoredlist` so you can see all the words you've ignored, and `/ignoreall` so you can add many at once, but other than that this is great, and thank you again!
